test/manual/css-skins/index.html

```diff
@@ -150,22 +138,22 @@
                 <div class="jw-controls">
                     <div class="jw-display">
                         <div class="jw-preview" style="background-image:url(http://content.bitsontherun.com/thumbs/3XnJSIm4-480.jpg)"></div>
-                        <div class="jw-display-icon-cont">
+                        <div class="jw-display-icon-container">
                             <div class="jw-text">TEST TEXT</div>
-                            <div class="jw-icon-display jw-icon-play"></div>
+                            <div class="jw-icon-display"></div>
                         </div>
                     </div>
                     <img class="jw-logo">
                     <div class="jw-dock">
                         <div class="jw-dock-button">
-                            <div class="jw-dock-image" style="background-image: url(../../../jwplayer-commercial/test/assets/os/both.png)"></div>
+                            <div class="jw-dock-image" style="background-image: url(/jwplayer-commercial/test/assets/os/both.png)"></div>
                             <div class="jw-arrow"></div>
                             <div class="jw-overlay">
                                 <span class="jw-text">Button sucessfully added</span>
                             </div>
                         </div>
                         <span class="jw-dock-button">
-                            <div class="jw-dock-image" style="background-image: url(../../../jwplayer-commercial/test/assets/smiley.png)"></div>
+                            <div class="jw-dock-image" style="background-image: url(/jwplayer-commercial/test/assets/smiley.png)"></div>
```

Comment:
move these assets to a location relative to test/manual/css-skins/ like test/manual/css-skins/icons. Then your test page can just point to icons/file